provisioning/standalone/vscode/c-cpp/Dockerfile

```diff
@@ -0,0 +1,43 @@
+# syntax = edrevo/dockerfile-plus
+
+#This stage can be used to compile locally the extension, can be useful with those releases which don't include a .vsix file
```

Comment:
```suggestion
# This stage can be used to compile locally the extension, and it can be useful with those releases which don't include a .vsix file
```